Exiled.API/Features/CustomHealthStat.cs

```diff
@@ -21,5 +21,13 @@ public class CustomHealthStat : HealthStat
         /// Gets or sets the maximum amount of health the player will have.
         /// </summary>
         public float CustomMaxValue { get; set; }
+
+        /// <inheritdoc/>
+        public override void ClassChanged()
+        {
+            base.ClassChanged();
+
+            CustomMaxValue = 0f;
```

Comment:
Should we remove `player::MaxHealth = default` in `ChangingRole::UpdatePlayerRole` method?

Reply:
Yeah